### PR TITLE
Plone 5 compatibility and bugfix

### PR DESCRIPTION
**Pull request by keul**
This is not here anymore on Plone 5 and it's automatically installed on Plone 4 (so removing this will not give any issue)

### CODE REVIEW

File: plone/formwidget/masterselect/static/masterselect.js [hunk 1]
```diff
@@ -1,221 +1,275 @@
...
+      }
+      options += sprintf(
+        '<option id="%s" value="%s"%s>%s</option>',
+        items[i].id,
+        items[i].value,
+        selected,
+        items[i].content
+      );
     }
-    function handleMasterVocabularyChange(event) {
```
Comment (tkimnguyen) [inline plone/formwidget/masterselect/static/masterselect.js]: this function isn't needed anymore?

**Comment by tkimnguyen** [comment]
I took a look even though I am definitely not the best suited person to be reviewing :) One thing I'd suggest is leaving style changes to a separate commit because it was harder to see the non-style changes you made.

**Comment by hvelarde** [comment]
do these changes make the package compatible with Dexterity in Plone 4?

**Comment by keul** [comment]
@hvelarde sorry don't know. I was not aware this was not working with Dx on Plone 4, I only found it was not working on Plone 5 due to dependency.

@tkimnguyen sorry probably I run a prettier before the commit... my changes are few lines.
I can try to revert the style changes.

**Comment by hvelarde** [comment]
@tkimnguyen the tests are passing; it should be fine to merge this.

**Comment by hvelarde** [comment]
@keul I think is better just to drop support for Python 2.6 and Plone 4.2.

**Comment by keul** [comment]
@hvelarde you are right, also because json module is there on Python 2.6

**Comment by hvelarde** [comment]
@keul you forgot to add an entry on the changelog explicitly stating: "Drop support for Python 2.6 and Plone 4.2."

**Comment by keul** [comment]
@hvelarde in facts my previous change/comment was wrong. Python 2.6 is already shipped with the `json` module so we are (probably) still compatible with Plone 4.2.

I think that simplejson was added when Plone 3 was supported

**Pull request merged**